style.css

```diff
@@ -0,0 +1,209 @@
+/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+
+body {
+  background-color: mintcream;
+  color: black;
+  margin: 0px;
+}
+ 
+
+h1 {
+  font-size: 50px;
+  font-family: 'Montserrat', sans-serif;
+  padding: 50px 10px;
+  margin: 15px 10px;
+  text-align: center;
+  }
+
+h2 {
+  font-size: 25px;
+  font-family: Nunito;
+  letter-spacing: 2px;
+  text-align: center;
+  font-weight: bold;
+  margin: 40px;
+  margin-bottom: 50px;
+}
+span {
+  color: #A39161;
+  font-weight: bold;
+}
+h3 {
+  font-size: 25px;
+  font-family: Nunito;
+  letter-spacing: 2px;
+  font-weight: bold;
+  text-align: center;
+  padding: 15px 10px;
+  margin: 15px 10px;
+}
+h4 {
+  font-size: 25px;
+  font-family: Nunito;
+  letter-spacing: 2px;
+  font-weight: bold;
+  text-align: center;
+}
+
+p {
+  font-size: 18px;
+  font-family: Nunito;
+  letter-spacing: 2px;
+  text-align: justify;
+  padding: 15px 10px;
+  margin: 15px 10px;
+}
+.container {
+  width: 100%;
+  max-width: 1100px;
+  margin: 0 auto; 
+  display: flex;
+  justify-items: center;
+} 
+  
+/*** Header ***/
+header {
+  background: #C9F4EB;
+  height: 100px;
+  display: flex;
+  align-items: center;
+  padding: 50px 16px;
+  background-image: url("./images/KH_header_backgr_a5_1.jpg");
+}
+
+.hamburger{
+  display: flex;
+  height: 30px;
+  width: 30px;
+  flex-direction: column;
+  justify-content: space-between;
+}
+
+.hamburger div {
+  width: 30px;
+  height: 5px;
+  background:black;
+  display: flex;
+ margin-left: 50px;
+  
+}
+
+nav {
+  display: none;
+  font-family: 'Darker Grotesque', sans-serif;
+  font-weight: bold;
+  margin: 0 5px;
+}
+
+nav a {
+  display: inline-block;
+  padding: 5px;
+  text-decoration: none;
+  border-radius: 5px;
+  font-size: 18px;
+  color: black;
+}
+
+nav a:hover {
+  background-color: white;
+  color: hotpink;
+}
+
+header img {
+  width: 200px;
+  margin: 15px;
+  
+}
+
+/*** INSPIRATION CONTAINER ***/
+
+.inspiration-container{
+  
+}
+
+.container {
+  display: flex;
+  background-color: white;
+  width: 90%;
+  height: 90%;
+  margin: 20px;
+  border-radius: 15px;
+  flex-direction: column;
+  align-items: center;
+}
+.container img {
+  max-width: 95%;
+  max-height: 95%;
+  padding: 5px 10px;
+  margin: 5px 10px;
+}
+.container:hover {
+  border: 2px solid #A39161;
+  border-radius: 15px;
+}
```

Comment:
Nice hover effect! Be aware though - that it makes the content jump a little bit because of the extra space that the border takes up. Consider adding a transparent border in the not-hovered-state to create that space in advance:
```
.container {
  border: 2px solid transparent;
}
```